pmm-app/tests/ia/pages/api/alertsAPI.js

```diff
@@ -0,0 +1,46 @@
+const { I } = inject();
+const assert = require('assert');
+
+module.exports = {
+  /*
+  waitForAlerts method is used for waiting for alerts to appear with a timeout.
+  expectedAlertsCount parameter is optional and only needed when we need to wait for a
+  specific number of alerts to appear
+  */
+  async waitForAlerts(timeout, expectedAlertsCount = 0) {
+    for (let i = 0; i < timeout; i++) {
+      const results = await this.getAlertsList();
+
+      // waiting for some exact number of alerts to be active
+      if (results && results.length >= expectedAlertsCount) break;
+
+      I.wait(1);
```

Comment:
nit: this is one millisecond, right? or one second?

Reply:
this is one second actually